rs/nns/governance/init/src/lib.rs

```diff
@@ -289,29 +294,17 @@ impl GovernanceCanisterInitPayloadBuilder {
 
             let neuron_id = NeuronIdProto::from(neuron_id);
 
-            let not_for_profit = record[5]
+            let not_for_profit = record[6]
                 .parse::<bool>()
                 .expect("couldn't read the neuron's not-for-profit flag");
 
-            let memo = if record.len() < 9 {
-                self.rng.next_u64()
-            } else {
-                record[6].parse::<u64>().expect("could not parse memo")
-            };
+            let memo = self.rng.next_u64();
```

Comment:
Why remove the ability to specify the memo?

Reply:
Despite this code still being here, the ability to specify memo has been removed in https://github.com/dfinity/ic/commit/b0ea17e4028c309ace18a69257268c05fd4b8ef2